model/validation.go

```diff
@@ -204,16 +204,367 @@ func (t Tags) Validate() error {
 	return errs
 }
 
+// Validate ensures tags are well-formed, but does not insist that tags exist
+func Validate(t map[string]string) error {
```

Comment:
There is a function `(t Tags) Validate()` that does the exact same. What's the purpose of code repetition? You may need to convert types with a function in `conversion.go` but the conversion does not require the map to be valid.